frontend/src/app/components/LandingPage/HeadingTexts.tsx

```diff
@@ -0,0 +1,20 @@
+import { DM_Serif_Display } from "next/font/google";
+
+const dm_Serif_Display = DM_Serif_Display({
```

Comment:
`DM_Serif_Display` is used in a bunch of different files and it could just be encapsulated into a simple import so you wouldn't need to write this over and over again

eg.
```
export dm_Serif_Display_latin = DM_Serif_Display({...})

export dm_Serif_Display_normal = DM_Serif_Display({...})
```